client/src/routes.ts

```diff
@@ -36,7 +36,6 @@ export const routes: Route[] = [
     unlisted: true
   },
   {
-    // TODO: add a button matching this route
```

Comment:
We could always move these buttons in the profile nav as well. I'm only saying that because conventionally, most websites don't have register/login on there main navigation. For instance, <https://github.com> 
![image](https://github.com/WSU-Society-of-Computer-Developers/summer-project/assets/35277113/ef72e1cf-3558-4cca-a3c5-f216c5652029)

Maybe this can be a backlog ticket too

Reply:
I agree. I'll make a ticket rn